src/lib/edge-functions/deploy.js

```diff
@@ -10,7 +9,7 @@ const distPath = getPathInProject([EDGE_FUNCTIONS_FOLDER])
 
 const deployFileNormalizer = (file) => {
   const isEdgeFunction = file.root === distPath
-  const normalizedPath = isEdgeFunction ? path.join(PUBLIC_URL_PATH, file.normalizedPath) : file.normalizedPath
```

Comment:
This effectively a URL, not a filesystem path. So it needs to use the UNIX format regardless of the host platform.